src/instrumenter/instrumentation_context.ts

```diff
@@ -113,6 +114,11 @@ export class InstrumentationContext {
         return this.utilsContract.parent as SourceUnit;
     }
 
+    /**
+     * Debug events assertion's signature
+     */
+    public debugEventsSignature: string;
```

Comment:
I think we can remove this field. Its always a constant, and it only seems to be used in json_output.ts. I think we can remove it from there as well (see comment in json_output.ts)